docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -114,11 +114,14 @@ The first approach is by providing the properties within the `src/main/resources
 quarkus.application.name=myservice // <1>
 quarkus.opentelemetry.enabled=true // <2>
 quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:55680 // <3>
+
+quarkus.opentelemetry.tracer.exporter.otlp.headers=Authorization=Bearer my_secret // <4>
 ----
 
 <1> All spans created from the application will include an OpenTelemetry `Resource` indicating the span was created by the `myservice` application. If not set, it will default to the artifact id.
 <2> Whether OpenTelemetry is enabled or not. The default is `true`, but shown here to indicate how it can be disabled
-<3> gRPC endpoint for sending spans.
+<3> gRPC endpoint for sending spans
+<4> Optional gRPC headers commonly used for authentication
```

Comment:
Documentation improvement. I didn't find an example in the docs to indicate that the param is optional so I just mentioned it in the textual explanation.